devito/core/gpu_openmp.py

```diff
@@ -306,6 +392,10 @@ def _specialize_iet(cls, graph, **kwargs):
         data_manager.place_definitions(graph)
         data_manager.place_casts(graph)
 
+        # Initialize OpenMP environment
+        initialize(graph)
+        mpi_ompizer(graph)
```

Comment:
This is the `DeviceOpenMPNoopOperator` -- the "noop" in the name implies that no optimizations are performed. So you probably wanna remove `mpi_ompizer` from here